### PR TITLE
Add invite button to bottom of top level settings screen

### PR DESCRIPTION
**Pull request by daehn**
# What's in this PR?

* Adds a sticky invite button to the bottom of the top level settings view.

### CODE REVIEW

**Comment by daehn** [comment]
Let me quickly run the tests…

**Comment by daehn** [comment]
All ✅

File: WireExtensionComponents/Views/OverflowSeparatorView.swift [hunk 1]
```diff
@@ -40,12 +44,20 @@ import Classy
...
     }
     
     public func scrollViewDidScroll(scrollView: UIScrollView!) {
-        self.alpha = scrollView.contentOffset.y > 0 ? 1 : 0
+        if !inverse {
+            self.alpha = scrollView.contentOffset.y > 0 ? 1 : 0
+        } else {
+            let (height, contentHeight) = (scrollView.bounds.height, scrollView.contentSize.height)
+            let offsetY = scrollView.contentOffset.y
+            print(contentHeight, offsetY, height, contentHeight - offsetY)
```
Comment (typfel) [inline WireExtensionComponents/Views/OverflowSeparatorView.swift]: print statement 😱

Comment (daehn) [inline WireExtensionComponents/Views/OverflowSeparatorView.swift]: I assume `Danger` is still asleep then…

**Comment by daehn** [comment]
@typfel updated!

**Pull request merged**